Makefile

```diff
@@ -82,6 +82,10 @@ ifeq ($(DISABLE_CGO), 1)
 	override BUILDTAGS = exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp
 endif
 
+ARCHES = amd64 s390x ppc64le
+MA_IMAGE := quay.io/skopeo/$(IMAGE_TYPE):ma
```

Comment:
IDK, but we may also want to push to quay.io/containers/skopeo:ma

Reply:
I see. I believe `quay.io/containers/skopeo:ma` should be based on the `stable` version, not on the `upstream`, which is target of this PR for now?
In general I'd suggest to add one more env variable - `EXTRA_REPO` (`EXTRA_REPOS?`) and when it exists - retag and push multi-arch image also to the extra quay.io repo(s).

Reply:
Yes, I'd suggest that the `quay.io/containers/skope:ma` should be based on the `stable` version.  Currently, `quay.io/containers/skopeo:latest` and `quay.io/skopeo/stable:latest` are equivalent.